crates/iota-light-client/Cargo.toml

```diff
@@ -5,35 +5,54 @@ authors = ["IOTA Foundation <info@iota.org>"]
 edition = "2021"
 license = "Apache-2.0"
 publish = false
+description = "A light client for the IOTA blockchain"
 
 [lib]
 path = "src/lib.rs"
 
 [[bin]]
 name = "iota-light-client"
-path = "src/bin/light_client.rs"
+path = "src/main.rs"
+
+[[bin]]
+name = "generate-chk-snapshots"
+path = "src/bin/generate_chk_snapshots.rs"
 
 [dependencies]
 # external dependencies
 anyhow.workspace = true
 async-trait.workspace = true
 bcs.workspace = true
+bin-version.workspace = true
 clap.workspace = true
 env_logger = "0.11.5"
+futures.workspace = true
+getset = "0.1"
 log = "0.4.22"
 object_store.workspace = true
+prometheus.workspace = true
 reqwest.workspace = true
+roaring.workspace = true
 serde.workspace = true
 serde_json.workspace = true
 serde_yaml.workspace = true
+telemetry-subscribers.workspace = true
+tempfile = "3.8.0"
```

Comment:
should be used from the workspace

Reply:
https://github.com/iotaledger/iota/pull/6135/commits/4ea494b6694422970e68076cff1f72f6075c115e

Reply:
isn't it already?